src/pages/home/HeaderView.js

```diff
@@ -175,96 +176,102 @@ function HeaderView(props) {
     const shouldShowBorderBottom = !isTaskReport || !props.isSmallScreenWidth;
     const shouldDisableDetailPage = ReportUtils.shouldDisableDetailPage(props.report);
 
+    const isLoading = !props.report || !title;
+
     return (
         <View
             style={[styles.appContentHeader, shouldShowBorderBottom && styles.borderBottom]}
             dataSet={{dragArea: true}}
         >
             <View style={[styles.appContentHeaderTitle, !props.isSmallScreenWidth && styles.pl5]}>
```

Comment:
Please retest after changes have been made
```suggestion
            <View style={[styles.appContentHeaderTitle, !isLoading && !props.isSmallScreenWidth && styles.pl5]}>
```